backend/src/main/resources/db/migration/V1_0_5__Add_Organization_Employees.sql

```diff
@@ -0,0 +1,18 @@
+DROP TABLE organization CASCADE;
```

Comment:
Why DROP?

Reply:
If I recall correctly, this statement was written because we could not delete the constraint related to this table in a proper way since the constraint ID was generated automatically by the DB and its value could vary between the local docker image and the one that Jenkins builds.
That said, it will be fixed before closing this branch to develop by merging all the flyway's changes into one single script as you said in the other comments.

Reply:
Ok, understood.
In any case we should try to generate those "constraint ID"s ourselves whenever is possible so that we have full control of any DB aspect in case of needing to write a rollback script. Dropping a table which could potentially hold data stored in it, is not something we can do once we are in a prod environment. Please review which constraints could not be undone so that they are created properly.